gstools/covmodel/models.py

```diff
@@ -117,14 +123,20 @@ class Exponential(CovModel):
 
     Notes
     -----
-    This model is given by the following variogram:
+    This model is given by the following variogram [1]_:
 
     .. math::
        \gamma(r)=
        \sigma^{2}
        \left(1-\exp\left(-s\cdot\frac{r}{\ell}\right)\right)+n
 
     Where the standard rescale factor is :math:`s=1`.
+
+    References
+    ----------
+    .. [1] Webster, R. and Oliver, M. A.
+           "Geostatistics for environmental scientists.",
+           John Wiley & Sons. (2007)
```

Comment:
I just created an issue at sphinx, since their generated latex bibliography can't handle doublings: https://github.com/sphinx-doc/sphinx/issues/8918

But you are right, `Webster2007` would be a better identifier ;-)

Reply:
Done.